tools/latest-kernel-full-config.sh

```diff
@@ -10,82 +10,31 @@ bail() {
     exit 1
 }
 
-# Function to display usage information
-usage() {
```

Comment:
IMO, you still need the `usage` function, probably a bit simplified, at least to explain people that mistakenly ran the script `latest-kernel-full-config.sh` without the `Makefile` target.